java/src/test/java/org/rocksdb/DBOptionsTest.java

```diff
@@ -424,6 +424,26 @@ public void dbWriteBufferSize() {
     }
   }
 
+  @Test
+  public void setWriteBufferManager() throws RocksDBException {
+    try (final Options opt = new Options()) {
+      final Cache cache = new LRUCache(1 * 1024 * 1024);
+      final long bufferSize = 2000l;
+      final WriteBufferManager writeBufferManager = new WriteBufferManager(bufferSize, cache);
```

Comment:
Lets move the instantiation of Cache and writeBufferManager (lines 430 - 432) also into the try block. 
Same comment for the code change in OptionsTest as well.